selfdrive/car/chrysler/carstate.py

```diff
@@ -69,6 +69,7 @@ def update(self, cp, cp_cam):
     self.lkas_counter = cp_cam.vl["LKAS_COMMAND"]["COUNTER"]
     self.lkas_car_model = cp_cam.vl["LKAS_HUD"]["CAR_MODEL"]
     self.lkas_status_ok = cp_cam.vl["LKAS_HEARTBIT"]["LKAS_STATUS_OK"]
+    self.acc_faulted = cp.vl["ACC_2"]["ACC_FAULTED"] == 1
```

Comment:
let's add this to carState

Reply:
Will do here: https://github.com/commaai/openpilot/pull/24399